main.go

```diff
@@ -192,6 +192,11 @@ func prune(cli *client.Client, deathNote *sync.Map) (deletedContainers int, dele
 			log.Println(err)
 		} else {
 			for _, container := range containers {
+				_, isReaper := container.Labels["org.testcontainers.ryuk"]
```

Comment:
Could we move the string label to a constant and use it here?

Reply:
@matthewmcnew done at [5a31bc5](https://github.com/testcontainers/moby-ryuk/pull/53/commits/5a31bc5aaa7d7d4c8d75117015e47e7c7c28214a)